Cargo.toml

```diff
@@ -8,7 +8,7 @@ documentation = "https://starship.rs/guide/"
 edition = "2021"
 homepage = "https://starship.rs"
 # Keep `/` in front of `README.md` to exclude localized readmes
-include = ["src/**/*", "build.rs", "Cross.toml", "LICENSE", "README.md"]
+include = ["src/**/*", "build.rs", "Cross.toml", "LICENSE", "/README.md", "docs/.vuepress/public/presets/toml/"]
```

Comment:
What do we include the preset tomls for? I don't think people installing them with Cargo will know where to find them.

Reply:
This was the original request of #3580, the motivating issue for this PR. The idea is that users can grab the TOML presets out of the cargo package and merge them in over their current configurations, providing one way for them to stay up to date.

I don't think it's strictly better than a proper preset system, but given that nobody's stepped up to work on it (and I'm likely not going to be able to dedicate any time to it until the end of this year at the earliest) I think this is a reasonable mid-term solution.